vendor/vendor.json

```diff
@@ -313,11 +325,11 @@
 			"versionExact": "v9.7.0"
 		},
 		{
-			"checksumSHA1": "HfqZyKllcHQDvTwgCaYL1jUPmW0=",
+			"checksumSHA1": "5UH4IFIB/98iowPCzzVs4M4MXiQ=",
 			"comment": "v9.7.0",
 			"path": "github.com/Azure/go-autorest/autorest/validation",
-			"revision": "8c58b4788dedd95779efe0ac2055bb6a1b9b8e01",
-			"revisionTime": "2018-01-06T16:29:27Z",
+			"revision": "1e3943ee722538420f21945f4bc820347abe433d",
+			"revisionTime": "2018-02-15T17:42:55Z",
 			"version": "v9.7.0",
 			"versionExact": "v9.7.0"
```

Comment:
from what I can see this is from `Azure/go-autorest` v10 - could we downgrade this for the moment, since I don't think it's needed?

Reply:
I'm encountering an issue here. v10 includes the feature `Expose the polling URL for futures.` which is used by the sdk-for-go IoTHub models.go.

Reply:
taking a look into this - this is because SDKv14 is vendored in, downgrading to `v12.5` should fix this for the moment (we'll upgrade in the near future)